services/ui-src/src/views/DemoMeasure/questions/OtherPerformanceMeasure.tsx

```diff
@@ -0,0 +1,68 @@
+import * as QMR from "components";
+import * as Q from "./AdditionalNotes";
+import * as CUI from "@chakra-ui/react";
+import { useCustomRegister } from "hooks/useCustomRegister";
+import { Measure } from "measures/types";
+import React from "react";
+
+export const OtherPerformanceMeasure = () => {
+  const register = useCustomRegister<Measure.Form>();
+  const rates = [
+    {
+      label: "",
+      denominator: "",
+      numerator: "",
+      rate: "",
+      id: 1,
+    },
+  ];
+  const [showRates, setRates] = React.useState(rates);
+
+  return (
+    <QMR.CoreQuestionWrapper label="9. Other Performance Measure">
+      <QMR.TextArea
+        label="Describe the other methodology used:"
+        {...register("OtherPerformanceMeasure-Explanation")}
+      />
+      <CUI.Box marginTop={10}>
+        {showRates.map((item) => {
+          return (
+            <CUI.Stack my={10}>
+              <CUI.Heading fontSize="lg" fontWeight="600">
+                Describe the Rate:
+              </CUI.Heading>
+              <QMR.TextInput
+                label="For example, specify the age groups and whether you are reporting on a certian indicator:"
+                name={`OtherPerformanceMeasure-Rates.${item.id}.description`}
```

Comment:
```suggestion
                {...register(`OtherPerformanceMeasure-Rates.${item.id}.description`)}
```
Use ...register here instead of name
Same applies for the one below.

Reply:
This is no longer an issue. After a discussion with devs, we determined that using "name" is the only reasonable/viable solution in this instance